src/cardsavr/CardsavrHelper.ts

```diff
@@ -83,6 +83,8 @@ export class CardsavrHelper {
             if (!cardholder_data.first_name) cardholder_data_copy.first_name = card_data.first_name;
             if (!cardholder_data.last_name) cardholder_data_copy.last_name = card_data.last_name;
             if (!card_data.name_on_card) card_data.name_on_card = card_data.first_name + card_data.last_name;
+            const meta_key: string = createMetaKey(card_data, address_data.postal_code);
+            cardholder_data_copy.custom_data = { reporting_id: meta_key, cardsavr_user_data: { meta_key: meta_key } };
```

Comment:
I thought we wanted `{ reporting_id : X, cardsavr_card_data : { meta_key : X } }`

Reply:
You're right.  Nice catch.